worker/uniter/runner/jujuc/secret-add.go

```diff
@@ -12,9 +12,9 @@ import (
 	"github.com/juju/gnuflag"
 	"github.com/juju/names/v4"
 
-	"github.com/juju/juju/core/secrets"
-
 	jujucmd "github.com/juju/juju/cmd"
+	secretscmd "github.com/juju/juju/cmd/juju/secrets"
```

Comment:
Not sure that we should be importing cmd/juju in the worker package.
There's not really enough common in the code to justify it,